job_spec/ARIA_S1_GUNW.yml

```
@@ -1,38 +1,25 @@
 ARIA_S1_GUNW:
   required_parameters:
-    - reference
-    - secondary
+    - reference_date
+    - secondary_date
     - frame_id
   parameters:
-    reference:
+    reference_date:
       api_schema:
-        type: array
-        minItems: 1
-        maxItems: 3
-        example:
-          - S1A_IW_SLC__1SSV_20141127T015250_20141127T015318_003461_0040D8_A29D
-          - S1A_IW_SLC__1SSV_20141127T015314_20141127T015342_003461_0040D8_87BA
-        items:
-          description: The names of the Sentinel-1 SLC granules to use as reference scenes for InSAR processing
-          type: string
-          pattern: "^S1[AB]_IW_SLC__1S[SD]V"
-          minLength: 67
-          maxLength: 67
-          example: S1A_IW_SLC__1SDV_20250127T010136_20250127T010203_057623_07199D_4B63
-    secondary:
+        description: The date to find Sentinel-1 SLC granules to use as reference scenes for InSAR processing
+        type: string
+        maxLength: 10
+        minLength: 10
+        pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
```

Slightly more concise:
```suggestion
        pattern: "^\d{4}-\d{2}-\d{2}$"
```

---

Going with `format: date` so there will be no need for the pattern.